src/Compilers/CSharp/Portable/Parser/Lexer_StringLiteral.cs

```diff
@@ -77,7 +77,18 @@ private void ScanStringLiteral(ref TokenInfo info, bool inDirective)
             }
             else
             {
-                info.Kind = SyntaxKind.StringLiteralToken;
+                if (!inDirective && TextWindow.PeekChar() == 'u' && TextWindow.PeekChar(1) == '8')
```

Comment:
note: for raw strings, i allowed them to be parsed in directives, but then give a particular message that it isn't allowed.  up to you if you want to do the same.

Reply:
> note: for raw strings, i allowed them to be parsed in directives, but then give a particular message that it isn't allowed.  up to you if you want to do the same.

I do not believe it is worth the effort